gallery/src/components/NxDrawer/NxDrawerPage.tsx

```diff
@@ -0,0 +1,258 @@
+          {' '}<NxCode>NxFooter</NxCode> (optional). The <NxCode>NxDrawer</NxCode> is only compatible
+          within <NxCode>NxPage</NxCode> layout with global header (it is not supported in legacy layout).
+        </NxP>
+        <NxTile.Subsection>
+          <NxH3>Props</NxH3>
+          <NxTable>
+            <NxTable.Head>
+              <NxTable.Row>
+                <NxTable.Cell>Prop</NxTable.Cell>
+                <NxTable.Cell>Type</NxTable.Cell>
+                <NxTable.Cell>Required</NxTable.Cell>
+                <NxTable.Cell>Default</NxTable.Cell>
+                <NxTable.Cell>Details</NxTable.Cell>
+              </NxTable.Row>
+            </NxTable.Head>
+            <NxTable.Body>
+              <NxTable.Row>
+                <NxTable.Cell>className</NxTable.Cell>
+                <NxTable.Cell>string</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  Any <NxCode>className</NxCode> attributes passed in on
+                  the <NxCode>NxDrawer</NxCode> will be added to
+                  the dialog element.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>onCancel</NxTable.Cell>
+                <NxTable.Cell>Function (() =&gt; void)</NxTable.Cell>
+                <NxTable.Cell>Yes</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  <NxCode>NxDrawer</NxCode>'s callback function. It gets called when the user
+                  closes the dialog (when the user clicks outside the drawer, when the user clicks the close button,
+                  and when the user presses the Esc key).
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>variant</NxTable.Cell>
+                <NxTable.Cell>"narrow" | "normal"</NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell>"normal"</NxTable.Cell>
+                <NxTable.Cell>
+                  This prop specifies a style variant for the drawer. Currently, variants only differ in width.
+                  "normal" drawers are 520px wide, and "narrow" drawers are 348px wide.
+                </NxTable.Cell>
+              </NxTable.Row>
+              <NxTable.Row>
+                <NxTable.Cell>HTML <NxCode>&lt;dialog&gt;</NxCode> Attributes</NxTable.Cell>
+                <NxTable.Cell>
+                  <NxTextLink external href="https://reactjs.org/docs/dom-elements.html#all-supported-html-attributes">
+                    HTML Attributes
+                  </NxTextLink>
+                </NxTable.Cell>
+                <NxTable.Cell>No</NxTable.Cell>
+                <NxTable.Cell></NxTable.Cell>
+                <NxTable.Cell>
+                  NxDrawer supports any html attribute that's normally supported by
```

Comment:
Should it support the `open` attribute? I tend not to think so since we manage that internally (by calling `.show()`)

Reply:
> Should it support the `open` attribute? I tend not to think so since we manage that internally (by calling `.show()`)

Changed here: ac5ac9ca911ba95e2533a0754eb0a7499305b0bd

Reply:
> Changed here: https://github.com/sonatype/sonatype-react-shared-components/commit/ac5ac9ca911ba95e2533a0754eb0a7499305b0bd

Looks good, you can also remove it from the typescript type using the [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) utility type

Reply:
> > Changed here: [ac5ac9c](https://github.com/sonatype/sonatype-react-shared-components/commit/ac5ac9ca911ba95e2533a0754eb0a7499305b0bd)
> 
> Looks good, you can also remove it from the typescript type using the [`Omit`](https://www.typescriptlang.org/docs/handbook/utility-types.html#omittype-keys) utility type

Updated here: f40b46d7025bed1626b43d44d960d82fde2e1a7b